pywbem/_subscription_manager.py

```diff
@@ -1334,7 +1430,9 @@ def _create_subscription(self, server_id, dest_path, filter_path, owned):
         server = self._get_server(server_id)
 
         sub_path = CIMInstanceName(SUBSCRIPTION_CLASSNAME,
-                                   namespace=server.interop_ns)
+                                   namespace=server.interop_ns,
+                                   keybindings=[('Filter', filter_path),
+                                                ('Handler', dest_path)])
```

Comment:
The path is not honored by CreateInstance, so we should also add `namespace=server.interop_ns` to the two `CreateInstance()` calls further down in this method.

Reply:
Actually our CreateInstance client code will get it from the path if it exists. but I added it to the CreateInstance in any case.